app/workers/distribution_worker.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
-class DistributionWorker < ApplicationWorker
+class DistributionWorker
   include Sidekiq::Worker
 
   def perform(status_id)
     FanOutOnWriteService.new.call(Status.find(status_id))
   rescue ActiveRecord::RecordNotFound
-    info("Couldn't find the status")
+    true
```

Comment:
Just as a comment for future reference: we don't really care if this status is not found. For example, user deletes status right after sending it out. It might already be gone before this job gets executed. That's okay.